src/plugins/internalnotification/README.md

```diff
@@ -40,6 +40,10 @@ Cascading keys as `key` names are also supported.
     `system/elektra/modules/internalnotification/exports/handle`.
 - *key* Key to watch for changes.
 
+*Returns*
+
+1 on success, 0 otherwise
```

Comment:
Isn't this information already in API docu? What about putting a link there?

Reply:
Yes, it kind of is in the API docs: The elektraNotificationRegister* functions are in the notification API docs, but not the functions exported by the plugin.
Since application developers should use the notification API instead of the exported functions I will add a link there.